src/angularComponent/ngcOmniboxSelectionsDirective.js

```diff
@@ -5,12 +5,17 @@ export default function ngcOmniboxSelectionsDirective() {
       omnibox: '^^ngcOmnibox'
     },
     scope: true,
+    bindToController: {
+      selection: '<'
+    },
+    controller: () => {},
     compile(tElement) {
       const element = tElement[0];
       const tokens = element.firstElementChild;
 
       if (tokens) {
         tokens.setAttribute('ng-repeat', 'selection in omnibox.ngModel');
+        tokens.setAttribute('selection', 'selection');
```

Comment:
I'm quite lost on this, you'll have to explain it to me. Do the bindings just override each other on the controller every loop iteration?